test/unit/alphabet/nucleotide_test.cpp

```diff
@@ -210,59 +166,59 @@ TYPED_TEST(nucleotide, explicit_conversion)
 TEST(dna4_literals, vector)
 {
     dna4_vector v;
-    v.resize(5, dna4::A);
+    v.resize(5, 'A'_dna4);
```

Comment:
There is a test missing that single quote literals are really have the rank representation, i.e. `'A'_dna4` == `dna4{}.assign_rank(0)`.